packages/utils/src/index.ts

```diff
@@ -24,3 +24,4 @@ export * from './resolveStaticPath';
 export * from './createTimeoutPromise';
 export * from './redactUserPath';
 export * as xssFilters from './xssFilters';
+export * as deviceUtils from './deviceUtils';
```

Comment:
in the start of this package, it was inteded to be completely trezor agnostic. and it has been like this until now. 
but since we probably dont have a better place I guess we may break this rule

Reply:
I can create a new package `device-utils` if we agree on that

Reply:
I will do it. Will be better for preparation for R

Reply:
yes, I think it will not hurt anybody. A new package for that.

Reply:
done